LEGO1/lego/legoomni/src/actors/pizzeria.cpp

```diff
@@ -76,9 +76,22 @@ MxU32 PizzeriaState::FUN_10017d70()
 	return 0;
 }
 
-// STUB: LEGO1 0x10017da0
+// FUNCTION: LEGO1 0x10017da0
 MxResult PizzeriaState::Serialize(LegoFile* p_legoFile)
 {
-	// TODO
-	return LegoState::Serialize(p_legoFile);
+	LegoState::Serialize(p_legoFile);
```

Comment:
To get this function to 100%, copy the code of `LegoState::Serialize` in here instead of calling the function, then it will match

Reply:
Thanks for the input and the research! I'd had a similar idea for templates when I saw your comment. Also, thanks for the reminder to look at the beta, I'm considering putting some information in the READMEs regarding that. If that's okay for you, I'll open a ticket to refactor the remaining existing code to these templates.